int32 	face_code	# code with the face to change to 1 - happy, 2 - sad, 3 - neutral, 4 - success, 5 - looking
